src/Console/ConfigurationResolver.php

```diff
@@ -577,8 +577,8 @@ private function computeConfigFiles()
 
         if ($this->isStdIn() || 0 === \count($path)) {
             $configDir = $this->cwd;
-        } elseif (1 < \count($path)) {
-            throw new InvalidConfigurationException('For multiple paths config parameter is required.');
+        } elseif (1 < \count($path) && null === $this->options['rules']) {
```

Comment:
using `--path-mode=intersection` will not work properly after this change, as it will intersect with implicit `$path[0]` or default config

Reply:
Should probably add a test for that as a contributor such as myself would have no way to tell that implicit overlap.

Reply:
not having explicit config to be used creates a lot of issues, instead of covering each of non-working variants with test confirming it doesn't work, we forbidden to not provide the explicit config when problematic params combination was provided.
saying that, i'm always happy to merge more good tests.